Makefile

```diff
@@ -10,3 +10,8 @@ env:
 	git submodule init
 	git submodule update --init --recursive
 	mkdir -p build && cd build && cmake ..
+
+.PHONY: Makefile
```

Comment:
Not clear why we need this phony target. Maybe a comment?

Reply:
It is actually unclear to me too. It was copied from the sphinx Makefile which does similar forwarding. Removing it means it stops working...